notebooks/week6/02.refresh_monitor.py

```
@@ -0,0 +1,119 @@
+])
+
+response_schema = StructType([
+    StructField("predictions", ArrayType(DoubleType()), True),
+    StructField("databricks_output", StructType([
+        StructField("trace", StringType(), True),
+        StructField("databricks_request_id", StringType(), True)
+    ]), True)
+])
+
+
+inf_table_parsed = inf_table.withColumn("parsed_request", 
```

Ruff (0.7.0)

Trailing whitespace

Remove trailing whitespace

(W291)